code/datums/skills/uscm.dm

```diff
@@ -140,7 +140,7 @@ MILITARY NONCOMBATANT
 	)
 
 /datum/skills/crew_chief
-	name = "Dropship Crew Chief"
+	name = "Dropship Pilot"
 	skills = list(
 		SKILL_PILOT = SKILL_PILOT_TRAINED,
```

Comment:
I recommend giving the Dropship Pilot SKILL_PILOT_EXPERT so they can fill in for the Gunship Pilot in the event of an emergency or vacancy.